Extensions/Lighting/lightruntimeobject-pixi-renderer.js

```diff
@@ -36,7 +38,7 @@ gdjs.LightRuntimeObjectPixiRenderer = function (runtimeObject, runtimeScene) {
 
   /** @type {?PIXI.Mesh} */
   this._light = null;
-  this._updateMesh();
```

Comment:
Not a private function anymore, because it needs to be called for hot-reloading, other than that there shouldn't really be any instance of this function being called. What do you suggest @4ian?

Reply:
Well if it's called from `updateFromObjectData`, it can still be private no? :) `updateFromObjectData` is public, but not necessarily what it is using.

Reply:
Ah I may have confused the pixi renderer and the object. Should be public then, because you expect ALL renderers to have this if you're calling it externally.